src/main/java/io/codelex/enums/Exercise2/Move.java

```diff
@@ -0,0 +1,29 @@
+package io.codelex.enums.Exercise2;
+
+public enum Move {
+
+    ROCK("r"),
+    PAPER("p"),
+    SCISSORS("s");
+
+    private String move;
+
+    Move(String move) {
+        this.move = move;
+    }
+
+    public String getMove() {
+        return move;
+    }
+
+    public void setMove(String move) {
+        this.move = move;
+    }
+
+    public static Move getMove(String move) {
```

Comment:
īstenībā ir jau built in metode `Move.valueOf()` vienīgi met excpetion uz nepareizas vērtības, ko tad var handlot